### PR TITLE
Adding TVIPS tools

### PR DESCRIPTION
**Pull request by hattne**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hattne** [comment]
@conda-forge-admin, please ping team

File: recipes/tvips-tools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  home: https://cryoem.ucla.edu/microed
+  license: ISC
+  license_file: LICENSE
+  summary: "Convert diffraction images from TVIPS format to SMV"
+  description: |
+    The TVIPS tools convert sequences of related electron diffraction
+    images, or MicroED sweeps, recorded by Tietz Video and Image
+    Processing Systems (TVIPS) cameras to the Super Marty View (SMV)
+    format.
+  dev_url: https://github.com/gonenlab/microed-tools
```
Comment (carterbox) [inline recipes/tvips-tools/meta.yaml:49]: This URL is not valid. Maybe the repo is private or was moved?

**Comment by carterbox** [comment]
@conda-forge-admin, please restart ci

File: recipes/tvips-tools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+source:
+  url: https://cryoem.ucla.edu/downloads/{{ tarname }}-{{ version }}.tar.gz
+  sha256: 87b5416470e4fa93b08c97ae8c73f71552945058e7f3a963e64248530646403d
+  patches:
+    - 01-setenv.patch
+    - 02-version.patch
+    - 03-warnings.patch
+
+build:
```
Comment (carterbox) [inline recipes/tvips-tools/meta.yaml:17]:
```suggestion
build:
   ignore_run_exports:
   - libstdcxx-ng
   - libcxx
```

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/tvips-tools/meta.yaml (hunk 2 above)
Comment (xhochy) [reply recipes/tvips-tools/meta.yaml:17]: Either

```suggestion
build:
  ignore_run_exports_from:
   - {{ compiler('cxx') }}
```

or do you really need the C++ compiler here? Could you simply omit it from `run`

File: recipes/tvips-tools/meta.yaml (hunk 1 above)
Comment (hattne) [reply recipes/tvips-tools/meta.yaml]: Removed optional dev_url.

File: recipes/tvips-tools/meta.yaml (hunk 2 above)
Comment (hattne) [reply recipes/tvips-tools/meta.yaml:17]: CXX is needed, but not the C++ standard library (that is pretty rare, I suspect)

**Review by carterbox — APPROVED** [review]

**Pull request merged**